test/test_core.py

```diff
@@ -5873,34 +5855,19 @@ def test_fs_64bit(self):
 
   @requires_node
   @crossplatform
-  @parameterized({
-    '': ([],),
-    'nodefs': (['-DNODEFS', '-lnodefs.js'],),
-    'noderawfs': (['-sNODERAWFS'],),
-  })
-  def test_fs_symlink_resolution(self, args):
-    nodefs = '-DNODEFS' in args or '-sNODERAWFS' in args
-    if nodefs and WINDOWS:
-      self.skipTest('No symlinks on Windows')
-    if self.get_setting('WASMFS'):
-      if nodefs:
+  @with_all_fs
+  def test_fs_symlink_resolution(self):
+    nodefs = '-DNODEFS' in self.emcc_args or '-DNODERAWFS' in self.emcc_args
+    if nodefs:
+      if WINDOWS:
+        self.skipTest('No symlinks on Windows')
+      if self.get_setting('WASMFS'):
```

Comment:
So this happens if you do `test.runner wasmfs.test_fs_symlink_resolution_nodefs`?

Reply:
Why does this test need this but the others not?

Reply:
Maybe this skip could be moved into the decorators?

Reply:
> So this happens if you do `test.runner wasmfs.test_fs_symlink_resolution_nodefs`?

Yes, and actually I was looking that current CI setup and it looks like we do run a bunch of wasmfs tests:
https://github.com/emscripten-core/emscripten/blob/a9108f89b5d2bb1b0a219b7a2a167271a287cf32/.circleci/config.yml#L570-L622